glue/viewers/image/viewer.py

```diff
@@ -1,10 +1,12 @@
 import os
+from tkinter.messagebox import YESNO
```

Comment:
What is this import for? I don't see it used.

Reply:
My editor seems to randomly insert imports...

Reply:
Haha, wut? This is what you get for not using Emacs! 😉

Reply:
PyCharm also just tried to secretly import urllib when I was trying to use a `unit`. 😝